### DISCUSSION TITLE
03: CNN for regression problems

### DISCUSSION THREAD

**Discussion by AlessandroPela** [mrdbourke/pytorch-deep-learning#564] [Q&A] [answered]
I have a general question:
I am looking at the course chapter 03 and I see that every Computer vision example is related to classification problems.
What do I have to change if I am working with a Regression Problem?

**Comment by Perian-Yan** [comment]
Hi,
I guess you can just get rid of the softmax, treating the convolution layer as a normal nonlinear layer.

**Comment by Perian-Yan** [comment, marked as answer]
There are many references using CNN for regression. (I didn't go into details of them)
https://www.datatechnotes.com/2019/12/how-to-fit-regression-data-with-cnn.html
https://github.com/rsyamil/cnn-regression

Hope this helps

**Reply by vimaurya** [reply]
Thank you, this was helpful.

**Reply by mrdbourke** [reply]
Thank you for helping @Perian-Yan !
